src/NovelRT/Ecs/SystemScheduler.cpp

```diff
@@ -0,0 +1,206 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#include <NovelRT/Ecs/SystemScheduler.h>
+
+namespace NovelRT::Ecs
+{
+    SystemScheduler::SystemScheduler(uint32_t maximumThreadCount) noexcept : _entityCache(1),
```

Comment:
Indentation here is different to other constructors; it should be made consistent.